### DISCUSSION TITLE
Klipper screen macros

### DISCUSSION THREAD

**Discussion by Reddy3d** [Guilouz/Klipper-Flsun-Speeder-Pad#102] [Q&A]
Macros icona errore,https://youtu.be/TI4fP6B5fPU
Link video

**Comment by Guilouz** [comment]
That’s means you have not the correct config files (macros.cfg)

**Reply by Reddy3d** [reply]
How to solved this problema?

**Reply by Guilouz** [reply]
Just replace config files.

**Reply by Reddy3d** [reply]
with which, that stock?
or do I have to write it?

**Reply by Reddy3d** [reply]
you can help me?

**Reply by Guilouz** [reply]
With configurations of your printer you use.

**Comment by Reddy3d** [comment]
Another question
Flip display of speeder pad Is possibile?
90/180 degrees
:)


Il sab 20 mag 2023, 14:47 Cyril Guislain ***@***.***> ha
scritto:
